scripts/test-fallback-scsv.py

```diff
@@ -211,21 +326,45 @@ def main():
     conversations["record SSL3.4 hello SSL3.4"] = conversation
 
     # test with FALLBACK_SCSV
-    for place in (0, 1, 2):
+    if dhe:
+        positions = (0, 1, 2, 3)
+    else:
+        positions = (0, 1, 2)
+    for place in positions:
         conversation = Connect(host, port)
         node = conversation
-        ciphers = [CipherSuite.TLS_EMPTY_RENEGOTIATION_INFO_SCSV,
-                   CipherSuite.TLS_RSA_WITH_AES_128_CBC_SHA,
-                   CipherSuite.TLS_AES_128_GCM_SHA256]
+        if dhe:
```

Comment:
It is necessary to have this test both in DHE and RSA? It looks like what's it is testing (rejection in position) doesn't depend on what ciphersuite is negotiated in the end.